release.rst

```diff
@@ -0,0 +1,208 @@
+[BUG] Fixes reproducibility in EdgeSwapper and adds to docs (#945)
```

Comment:
So here's the part where it's going to get tricky: - this includes all of the commits, not just the title of the PR.  We're going to need to make sure we limit it to only the PR title, because that's the human readable one, and our commit logs are more like "jfc will you JUST WORK WHY IS IT THIS HARD".  life of a frustrated developer and all that.

Next thing is that we actually have to figure out how to prepend it into the release.rst that's inside of our [docs/reference/release.rst](https://github.com/microsoft/graspologic/blob/dev/docs/reference/release.rst) file.  We'll want to put it after the title but before the last release, and give it a header (probably based off the setup.cfg file).